data/names.yaml

```diff
@@ -248,7 +247,7 @@ names:
       mg: Asompsiona
       mk: Успение на Пресвета Богородица
       mt: Santa Marija
-      nl: O.L.V. Hemelvaart
+      nl: Maria-Tenhemelopneming
```

Comment:
For this one, I chose to keep the name that is used in Belgium (see e.g. https://www.belgium.be/nl/over_belgie/land/belgie_in_een_notendop/feestdagen), as it's a national holiday there whereas in the Netherlands it isn't officially recognized (as far as I know). It's tough because the Netherlands and Belgium both use Dutch, but we disagree on the names of some of these holidays.

Reply:
thanks, reverted